### ISSUE TITLE
live webcam command does not work:  "Video stream has stopped."

### ISSUE THREAD

**Issue by stephanecharette** [hank-ai/darknet#38]
Attempting to run this command:  `darknet detector demo cfg/coco.data cfg/yolov4-tiny.cfg yolov4-tiny.weights -c 0`

Instead of it showing the webcam, console messages are logged:

```
Video stream has stopped.
Video stream has stopped.
Video stream has stopped.
Video stream has stopped.
...etc...
```

**Comment by stephanecharette** [comment]
A Darknet-enabled webcam application has since been added to the DarkHelp CLI.

See:  https://www.ccoderun.ca/darkhelp/api/Cam.html

**Comment by stephanecharette** [comment]
This should be fixed in v2.0.222, hash #f072df1.

(But I still recommend the one in DarkHelp since it has more features than using Darknet directly.)
